spotinst/organization_programmatic_user/consts.go

```diff
@@ -0,0 +1,15 @@
+package organization_programmatic_user
+
+import "github.com/spotinst/terraform-provider-spotinst/spotinst/commons"
+
+const (
+	Name             commons.FieldName = "name"
+	Description      commons.FieldName = "description"
+	Accounts         commons.FieldName = "accounts"
+	AccountId        commons.FieldName = "account_id"
+	AccountRole      commons.FieldName = "account_role"
+	Policies         commons.FieldName = "policies"
+	PolicyAccountIds commons.FieldName = "policy_account_ids"
+	PolicyId         commons.FieldName = "policy_id"
+	UserGroupIds     commons.FieldName = "user_group_ids"
```

Comment:
why is this field is under this resource

Reply:
we have kept it under this resource to allow the user to update userGroupIds mapping...